doc/system_requirements.md

```diff
@@ -4,73 +4,130 @@ System Requirement Specification OpenFastTrack
 
 # Introduction
 
+OpenFastTrack (OFT) is a requirement tracing suite written in Java.
+
+## Goals
+
+The OFT's goals are to be
+
+  * Fast
+  * Automation friendly
+  * Useful
+  * Platform independent
+
+A requirement tracing suite must be fast in order to be used. Development time is precious and requirement tracing does not have a high priority compared to building and testing the software regularly. This means that tracing will only be accepted by the users if it does not eat away their development time.
+
+Requirement tracing is a recurring and frankly quite boring task best performed by a machine instead of a person. Again the chances of getting accepted depend on getting this job "out of the way" and this means automating it. OFT therefore aims to offer easy integration into existing build processes, independently of the tool chains used.
+
+It looks strange first that "useful" needs to be spelled out as a goal. The reason is that the authors felt that existing tracing tools are mostly optimized for producing proof that you worked according to process - i.e. reports that you can shelve until an quality auditor wants to see them. While OFT can do this to, it is not a main goal. Instead OFT wants to help developers and technical writers to find gaps and mismatches in their requirement coverage fast and with as little effort as possible.
+
+OFT aims to help developers independently of their platform. I should not matter if you are developing on Linux, a Mac, BSD, Windows. Also it should not matter which programming language or development environment you are using. OFT aims to be portable and provide interfaces that allow integration into your existing toolchain. 
+
 ## Terminology
 
+  * Coverage provider: a specification item that provides coverage
```

Comment:
Fixed. See chapter Terms and Abbreviations (detail subsections).
Also added new terms to the list.